dash/frontend/src/app/modules/private/pages/cluster/kube-bench/kube-bench.component.html

```diff
@@ -122,7 +122,7 @@ <h3 class="error" *ngIf="statusInvalid">{{benchmarkStatus}}</h3>
                     </ng-container>
 
                     <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
-                    <mat-row class="bench-audit-row" *matRowDef="let row; columns: displayedColumns" (click)="toReportDetails(row.id)"></mat-row>
+                    <mat-row class="bench-audit-row" *matRowDef="let row; columns: displayedColumns" (click)="toReportDetails(row.id)"  [ngStyle]="{'cursor': 'pointer'}"></mat-row>
```

Comment:
I wonder if we should create a global "clickable" class or something for this kind of thing? @becky-intelletive  @kyle-intelletive @charisprose

Reply:
We have a global CSS class called 'pointer' that applies the 'cursor: pointer' rule.

We should also not use inline styles from a best practices & maintainability standpoint